### PR TITLE
Add a simple contribution guide

### PR DESCRIPTION
**Pull request by LinuxSuRen**
I'm not familiar with front-end development. But it works for me at least. Please don't hesitate to let me know if there's anything that I can improve.

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Welcome to join us. Please read the following instructions to setup your local development environment.
+
+First, install `pnpm` according to [the official document](https://pnpm.io/installation).
+
+then, install some necessary depdencies:
+
+```shell
+pnpm install vite
```
Comment (Dreamacro) [inline CONTRIBUTION.md:8]: `pnpm install` or `pnpm i` is enough, it installs all the required dependencies

**Review by Dreamacro — CHANGES_REQUESTED** [review]

File: CONTRIBUTION.md (hunk 1 above)
Comment (LinuxSuRen) [reply CONTRIBUTION.md:8]: Thanks for pointing this. I have tested it with `pnpm install`, it works well.

**Review by Dreamacro — APPROVED** [review]

**Comment by Dreamacro** [comment]
Thanks for contribution 👏

**Comment by LinuxSuRen** [comment]
Happy to help on this project.

**Pull request merged**